src/storage/mvcc/lock.rs

```diff
@@ -84,6 +91,9 @@ impl Lock {
             b.push(v.len() as u8);
             b.extend_from_slice(v);
         }
+        if self.is_pessimistic_txn {
+            b.push(PESSIMISTIC_TXN);
+        }
```

Comment:
We already have lock_type = PessimisticLock why we need `b.push(PESSIMISTIC_TXN);` ?

Reply:
It's different.
The lock type has been changed to PUT/DELETE，but we need to know it is written by a pessimistic transaction.

Reply:
I need a way to distinguish pessimistic txns from optimistic txns. If a pessimistic lock is prewrited it becomes a `normal` lock that It can't be distinguished.